src/com_tjvendors/admin/tjvendorsTemplate.json

```diff
@@ -92,27 +120,41 @@
     "client": "com_tjvendors",
     "key": "createVendorMailToAdminWaitingForApproval",
     "title": "Vendor Creation email to Admin (approval On)",
-    "email_body": "<p>Dear Admin,<br /><br />A new vendor named <strong>{info.vendor_title}</strong> has registered on the site by {vendorer.name}. <a href='{info.allVendors}'>Click here</a> to see all the vendors.<br />Note that you need to approve this vendor before they go live on the site.<br /><br />This is system generated email, please do not reply. Many thanks from {info.sitename}.</p>",
-    "sms_body": "",
-    "push_body": "",
-    "web_body": "",
-    "email_subject": "Vendor \"{info.vendor_title}\" needs approval at {info.sitename}",
-    "sms_subject": "",
-    "push_subject": "",
-    "web_subject": "",
-    "email_status": 1,
-    "sms_status": 0,
-    "push_status": 0,
-    "web_status": 0,
     "user_control": 1,
+    "state": 1,
     "core": 1,
+    "email":{
+		"state": 1,
+		"cc": "",
+		"bcc": "",
+		"from_name": "",
+		"from_email": "",
+		"emailfields": {
+			"emailfields0": {
+				"id": "",
+				"language": "*",
+				"subject": "Vendor \"{{info.vendor_title}}\" needs approval at {{info.sitename}}",
+				"body": "<p>Dear Admin,<br /><br />A new vendor named <strong>{{info.vendor_title}}</strong> has registered on the site by {{vendorer.name}}. <a href='{{info.allVendors}}'>Click here</a> to see all the vendors.<br />Note that you need to approve this vendor before they go live on the site.<br /><br />This is system generated email, please do not reply. Many thanks from {{info.sitename}}.</p>"
+			}
+		}
+	},
+	"sms": {
+		"state": 1,
+		"smsfields": {
+			"smsfields0": {
+				"id": "",
+				"language": "*",
+				"body": "Dear Admin, A new vendor named {{info.vendor_title}} has registered on the site by {{vendorer.name}}. Click here to see all the vendors. Note that you need to approve this vendor before they go live on the site."
```

Comment:
Change as 

Dear Admin, A new vendor {info.vendor_title} has registered on the site by {vendorer.name}. Visit admin panel to approve this vendor on site.